### DISCUSSION TITLE
QEditor remove link from popup

### DISCUSSION THREAD

**Discussion by ivankorobchuk123** [quasarframework/quasar#15575] [Quasar App Extensions]
How to remove the text in the link when the popup closes?
<img width="781" alt="Screenshot 2023-03-16 at 19 36 17" src="https://user-images.githubusercontent.com/24935631/225737603-13f2d19d-a63f-4a0e-a73e-26936af563bf.png">

**Comment by MilosPaunovic** [comment]
Duplicate #15574
